RecoTracker/TkTrackingRegions/src/OuterHitPhiPrediction.h

```diff
@@ -4,32 +4,40 @@
 /** predicts phi range at a given radius r */
 
 #include "RecoTracker/TkMSParametrization/interface/PixelRecoRange.h"
-#include "RecoTracker/TkTrackingRegions/interface/TkTrackingRegionsMargin.h"
 
 #include "FWCore/Utilities/interface/Visibility.h"
 
+#include <cassert>
+
 class dso_internal OuterHitPhiPrediction {
 public:
 
-  typedef PixelRecoRange<float> Range;
-  typedef TkTrackingRegionsMargin<float> Margin;
+  using Range= PixelRecoRange<float>;
 
   OuterHitPhiPrediction( 
       const Range & phiAtVertex, 
       const Range & curvature, 
-      float originRBound, 
-      const Margin & tolerance = Margin(0,0))
+      float originRBound) 
     : thePhiAtVertex(phiAtVertex), theCurvature(curvature),
-      theOriginRBound (originRBound), theTolerance(tolerance) { } 
+      theOriginRBound (originRBound) {
+       assert(theCurvature.max()>0);
```

Comment:
@Martin-Grunewald I beg your pardon, actually `80X_dataRun2_HLT_relval_v11` has later snapshot time than `80X_dataRun2_HLT_frozen_v12` (because it was created later in time):

https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/80X_dataRun2_HLT_frozen_v12/80X_dataRun2_HLT_relval_v11

even if on the surface only the L1 menu tag is different, you can see in the table called `IOV differences` that many (actually 51) tags have developed new IOVs that are not seen in `80X_dataRun2_HLT_frozen_v12`  because they were inserted after its snapshot time.
Can you try with the "open" GT that goes directly online, I mean: `80X_dataRun2_HLT_v12`  ?